resubmit.py

```diff
@@ -251,9 +209,9 @@ def extendWorkflow(workflow, user, group, verbose=False, events=None, firstlumi=
     firstlumi = int(firstlumi)
 
     # Get info about the workflow to be cloned
-    helper = reqMgrClient.retrieveSchema(workflow)
-    schema = modifySchema(helper, workflow, user, group, None, events, firstlumi, None)
-    schema['OriginalRequestName'] = workflow
+    cache = reqMgrClient.getWorkflowInfo(url, workflow)
```

Comment:
Maybe it was the merge* keys? But it should be present in all requests for several months now